packages/stratimikos/test/_FourByFour.belos.ifpack.xml

```diff
@@ -2,6 +2,16 @@
   <Parameter name="Matrix File" type="string" value="FourByFour.mtx"/>
   <ParameterList name="Linear Solver Builder">
     <Parameter name="Linear Solver Type" type="string" value="Belos"/>
+    <ParameterList name="Linear Solver Types">
+      <ParameterList name="Belos">
+        <Parameter name="Solver Type" type="string" value="Pseudo Block GMRES"/>
+        <ParameterList name="Solver Types">
+          <ParameterList name="Pseudo Block GMRES">
+            <Parameter name="Convergence Tolerance" type="double" value="1.0e-16"/>
```

Comment:
I'm glad the tolerance isn't -1 any more. Does it really need to be less than machine precision, though?

Reply:
Probably not. But this way the test will more or less do what it did before, i.e. solve to machine precision. Once this unblocks other PRs, we should probably check what we are actually testing here..

Reply:
I only mention it because if we're going to get the PR through anyway, we should at least make it some small integer factor times machine precision.

Reply:
Fair enough, let's make sure this will actually work.